pkgs/by-name/xr/xrsh/package.nix

```
@@ -0,0 +1,64 @@
+{
+  stdenv,
+  fetchFromGitea,
+  makeWrapper,
+  writeShellApplication,
+  python3Packages,
+  lib,
```

Apologies if I wasn't clear, but the updater also needs to be added as an input 😁

```suggestion
  lib,
  unstableGitUpdater,
```

---

done